pkgs/top-level/default.nix

```diff
@@ -61,10 +61,23 @@ in let
   localSystem = lib.systems.elaborate args.localSystem;
 
   # Condition preserves sharing which in turn affects equality.
+  #
+  # See `lib.systems.equals` documentation for more details.
+  #
+  # Note that it is generally not possible to compare systems as given in
+  # parameters. E.g. system is initialized to
+  #
+  #   localSystem.system = "x86_64-linux";
+  #   crossSystem.config = "x86_64-unknown-linux-gnu";
+  #
+  # when using `legacyPackages.x86_64-linux.pkgsCross.gnu64` from flakes.
```

Comment:
I don't think it is good to mention flakes here. We should be talking about what sort of arguments this thing receives in isolation, not who is calling it (flakes or otherwise).

Reply:
Done.

Reply:
Thanks